### ISSUE TITLE
Display box for DAI searches not displaying search criteria (as it did previously)

### ISSUE THREAD

**Issue by cneidle** [DCS-LCSR/ASL-DAI#62]
I assume that this is a feature that broke when you changed the template, to use Christian's.  But previously, the upper left-hand box showed what had been searched for.  In the example below, I searched for fingerspelled signs, but this information is missing on top.  The problem is considerably more general.  Again, this used to work much better.  (sorry)

![screen shot 2018-01-31 at 11 17 21 am](https://user-images.githubusercontent.com/13629122/35633873-55198580-0678-11e8-8654-b64f8f175c95.png)

**Comment by kwasiopoku** [comment]
Fixed

**Comment by cneidle** [comment]
Some of these work; some don't.

The types of signs are not shown.

<img width="542" alt="screen shot 2018-02-19 at 6 58 29 pm" src="https://user-images.githubusercontent.com/13629122/36401752-910e203a-15a7-11e8-8f3c-f9b2b19c7dd3.png">

<img width="410" alt="screen shot 2018-02-19 at 6 58 39 pm" src="https://user-images.githubusercontent.com/13629122/36401758-9ae2a040-15a7-11e8-8202-b503ab70f6a5.png">

Selections of participants of specific collections are not shown.


<img width="255" alt="screen shot 2018-02-19 at 7 01 41 pm" src="https://user-images.githubusercontent.com/13629122/36401778-bb7d4b48-15a7-11e8-933f-044bfb0d0dd3.png">

Also I searched for a handshape.  Then returned to the main page and did "clear all".
But when I did the search, the handshape remained selected on the results page (although it was no longer visible on the main search page).   It did not clear it  from the "memory" of the search...
